randovania/games/samus_returns/logic_database/Area 5 Tower Interior.txt

```diff
@@ -474,6 +449,38 @@ Extra - asset_id: collision_camera_007
                       Movement (Advanced)
   > Event - Grapple Block
       Grapple Beam
+  > Below Pickup
+      Screw Attack
+
+> Below Pickup; Heals? False
+  * Layers: default
+  > Door to Plasma Beam Chamber
+      Grapple Beam and Morph Ball and After Area 5 (Tower Interior) - Grapple Shuffler Grapple Block
```

Comment:
why does this need grapple if the grapple block is already gone?

Reply:
Oops this is supposed to say Screw Attack, not Grapple.